base_user_effective_permissions/models/res_users_effective_permission.py

```diff
@@ -49,21 +49,17 @@ def _generate_permissions(self, user):
                 .with_context(allowed_company_ids=user.company_id.ids)
             )
             vals = {"model_id": model_record.id}
-            vals.update(
-                {
-                    f"{operation}_permission": model.check_access_rights(
-                        operation, False
-                    )
-                    for operation in operations
-                }
-            )
-            vals.update(
-                {
-                    f"{operation}_domain": IrRule._compute_domain(
+            for operation in operations:
+                try:
+                    model.check_access(operation)
```

Comment:
this is not the equivalent of v16's `model.check_access_rights` - use `ir.model.access#check` instead